main/lsp/LSPIndexer.cc

```diff
@@ -310,6 +319,12 @@ LSPFileUpdates LSPIndexer::commitEdit(SorbetWorkspaceEditParams &edit, std::vect
     if (!update.canTakeFastPath) {
         update.updatedGS = initialGS->deepCopy();
         pendingTypecheckUpdates = update.copy();
+        if (pendingTypecheckLatencyTimer != nullptr) {
+            pendingTypecheckLatencyTimer->cancel();
+        }
+        if (latencyTimer != nullptr) {
```

Comment:
I might make `latencyTimer` non-null in a future update; it adds a lot of noise to the code.